Launcher/Classes/ViewControllers/InspectorViewController.swift

```diff
@@ -19,6 +19,7 @@ class InspectorViewController: NSViewController, NSTableViewDataSource {
     private let defaultGestureRecognizerAccessibilityLabel = "defaultImage"
     private let customGestureRecognizerAccessibilityLabel = "customImage"
     private let elementInspectorPath = "/tmp/get_all_elements_inspector.txt"
+    private let cloneInfoPath = "/tmp/clone_info.txt"
```

Comment:
I was thinking about this - maybe in this `ViewController`, we should create an internal `Constants` enum to group these. :)